### PR TITLE
Our Changes to the Sampler for the JPro-Demo-Page

### PR DESCRIPTION
**Pull request by FlorianKirmaier**
We've made some changes to the project to promote it on the [JPro-Demo-Page](https://www.jpro.one/?page=demos).
It's accessable from the following url: [https://demos.jpro.one/bootstrapfx.html](https://demos.jpro.one/bootstrapfx.html)

It contains the following changes:
* Added a link to the online-demo at jpro.one.
* Added an ambient background to the sampler.
* The sampler now also works as a jar. Previously, it couldn't properly load the text of the fxml-files.

### CODE REVIEW

**Comment by aalmiray** [comment]
Thank you for the PR, however I'd suggest creating a separate sampler subproject that deals with JPro specifics. What do you think?

**Comment by FlorianKirmaier** [comment]
You mean as a subproject, which inherits all classes from the original sampler, but contains all the JPro-specific code?

I can do that.

**Comment by aalmiray** [comment]
Yes. Perhaps a refactor of the existing sampler is needed in order for the JPro sampler to reduce duplication when inserting the top level container. WDYT?

**Comment by FlorianKirmaier** [comment]
@aalmiray
I've moved the JPro-specific code to an own subproject.
I think it's now quite a bit cleaner.

File: README.adoc [hunk 1]
```diff
@@ -148,6 +148,11 @@ $ ./gradlew build
 [source]
 ----
 $ ./gradlew :sampler:run
+----
+ . Run the sampler application with JPro (https://www.jpro.one/) by invoking the following command
+[source]
+----
+$ ./gradlew :sampler:run
```
Comment (aalmiray) [inline README.adoc]: should this be `:sample-jpro:run` ?

File: subprojects/sampler-jpro/src/main/java/org/kordamp/bootstrapfx/SamplerJPro.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.kordamp.bootstrapfx;
+
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.layout.StackPane;
+import javafx.stage.Stage;
+
```
Comment (aalmiray) [inline subprojects/sampler-jpro/src/main/java/org/kordamp/bootstrapfx/SamplerJPro.java:28]: I'd encourage you to add your name as `@author` 😏

**Review by aalmiray — APPROVED** [review]

**Pull request merged**